build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -885,6 +891,9 @@ def iree_generated_e2e_matmul_test(
         generator_args_block = self._convert_string_list_block(
             "GENERATOR_ARGS", generator_args
         )
+        # Use the fully qualified target name
+        target_name_prefix = "iree_tools_testing_e2e_"
+        test_runner = f"{target_name_prefix}{test_runner.split(':')[-1]}"
```

Comment:
I thought we had a generic helper function for this target remapping. Would prefer to not code a specific path in here.

Reply:
Totally makes sense.